.github/ISSUE_TEMPLATE/task-module-service.md

```diff
@@ -0,0 +1,26 @@
+---
+name: "Task: NixOS service module"
+about: Serivce module task for an NGI project
+title: "Implement NixOS service module for <PROJECT_NAME>"
+labels: ''
+projects: Nix@NGI
+assignees: ''
+---
+
+### Instructions
+
+<!-- Replace <PROJECT_NAME> in the title and the body of this issue with the project's name. -->
+
+<!-- Replace `<TRIAGE_ISSUE_NUMBER>` with the issue number that contains the project's triaged information.
+If one doesn't exist, create it by following the instructions in the [contributor documentation](https://github.com/ngi-nix/ngipkgs/blob/main/CONTRIBUTING.md#triaging-an-ngi-project). -->
+
+Given #<TRIAGE_ISSUE_NUMBER> for metadata about the application, implement a NixOS service module for <PROJECT_NAME>.
+
+### Tasks
+
+After or during the implementation of this module, create issues for the following deliverables using the appropriate GitHub template:
+
+- [ ] Usage example configuration
```

Comment:
It's either

```suggestion
- [ ] Example configuration
```

or

```suggestion
- [ ] Usage example
```

But I see what you tried to do there. Unless you want to go for a bit of a linguistic stretch, probably example config is more suitable here because it better indicates that this is something testable. Unless you want to emphasize it's about the using. In any case since this is mostly for internal communication either is fine, and we'll fix forward as the need arises.